main/webapp/modules/core/scripts/reconciliation/recon-manager.js

```diff
@@ -63,6 +63,14 @@ ReconciliationManager.registerService = function(service) {
 };
 
 ReconciliationManager.registerStandardService = function(url, f, silent) {
+
+  if (ReconciliationManager._urlMap.hasOwnProperty(url)) {
+    if (!silent) {
+      alert($.i18n('core-recon/url-already-registered'));
+    }
+    return;
```

Comment:
Instead of just returning we might need to call the callback `f` if it is defined (see further down in the function).

Reply:
Thanks for the quick response, I made the requested modifications. Hope I understood correctly uwu

Reply:
Yes indeed, we still need the return statement in addition to calling the callback (see below)